daemon/mgr/container_storage.go

```diff
@@ -173,22 +173,29 @@ func (mgr *ContainerManager) getMountPointFromBinds(ctx context.Context, c *Cont
 			// volume bind.
 			name := mp.Source
 			if _, exist := volumeSet[name]; !exist {
-				mp.Name = name
-				mp.Source, mp.Driver, err = mgr.attachVolume(ctx, name, c)
+				_, mp.Driver, err = mgr.attachVolume(ctx, name, c)
 				if err != nil {
 					logrus.Errorf("failed to bind volume(%s): %v", name, err)
 					return errors.Wrap(err, "failed to bind volume")
 				}
 
-				volumeSet[mp.Name] = struct{}{}
+				volumeSet[name] = struct{}{}
 			}
 
-			if mp.Replace != "" {
-				mp.Source, err = mgr.VolumeMgr.Path(ctx, name)
-				if err != nil {
-					return err
-				}
+			volume, err := mgr.VolumeMgr.Get(ctx, name)
+			if err != nil || volume == nil {
+				logrus.Errorf("failed to get volume: (%s), err: (%v)", name, err)
```

Comment:
Could you use the same way to construct log and error message? 
I found that in L178, you use `logrus.Errorf("failed to bind volume(%s): %v", name, err)`, but here `logrus.Errorf("failed to get volume: (%s), err: (%v)", name, err)`.
How about that?

Reply:
fixed